tests/Application/public/.well-known/apple-developer-merchantid-domain-association

```diff
@@ -0,0 +1 @@

```

Comment:
I'm not sure if this file should be committed to the repository

Reply:
It is a certificate from Tpay (available publicly to download). So it's totally fine to store it in VCS